.circleci/config.yml

```diff
@@ -85,7 +85,7 @@ jobs:
             rm -rf /var/lib/apt/lists/*
       - run: gcc --version
       - checkout
-      - run: git submodule update --init --recursive --remote
+      - run: git submodule update --init --recursive

```

Comment:
We shouldn't be building non-reproducible builds - the remote setting means use the latest build, which we don't know what it is after building.  Would much rather explicitly update the git extern with specific versions.